platform/mk3/capabilities.h

```diff
@@ -17,9 +17,9 @@
 #define WIFI_SUPPORT		1
 
 /* Wifi Specific Info */
-#define WIFI_MAX_BAUD		230400
-#define WIFI_MAX_SAMPLE_RATE	10
-#define WIFI_ENABLED_DEFAULT	false
+#define WIFI_MAX_BAUD       921600
```

Comment:
Well... be careful here as this is how I found the limitations of the ESP8266 before.  YMMV but if you start seeing dropped characters remember your baud settings.